docs/source/model_doc/mbart.rst

```diff
@@ -76,6 +76,65 @@ the sequences for sequence-to-sequence fine-tuning.
     assert translation == "Şeful ONU declară că nu există o soluţie militară în Siria"
 
 
+MBart-50
+-----------------------------------------------------------------------------------------------------------------------
+
+mBART-50 is created using the original mBART model and extended to add extra 25 languages to support multilingual
```

Comment:
```suggestion
mBART-50 is created using the original mBART model and extended to add extra 25 languages to support multilingual
```

This is a bit difficult to understand IMO. It would be nice if we can be more precise here, maybe:

`mBART-50` is based on the original mBART model architecture. Compared to mBART-large-cc25, mBART-50 was  pretrained on 25 additional languages to support a total of 50 languages for machine translation (is the model only used for machine translation?)

Reply:
Yes, it's a bit confusing, what I meant by extended is that the authors actually used the `mBART-large-cc25` and resized the embedding and further pre-trained it on additional languages